cpp-ch/local-engine/Parser/CHColumnToSparkRow.cpp

```diff
@@ -809,8 +809,11 @@ int64_t VariableLengthDataWriter::writeStruct(size_t row_idx, const DB::Tuple &
                 writer.unsafeWrite(reinterpret_cast<const char *>(&v), buffer_address + offset + start + len_null_bitmap + i * 8);
             }
             else
-                writer.unsafeWrite(
-                    reinterpret_cast<const char *>(&field_value.get<char>()), buffer_address + offset + start + len_null_bitmap + i * 8);
+            {
+                const char * src = writer.getWhichDataType().isFloat64() ? reinterpret_cast<const char *>(&field_value.get<Float64>())
```

Comment:
Why?

Reply:
Let's fix it in another pr.  It annoy me in the debug build where calling get<char> for Float64 type throw an exception.

Reply:
add https://github.com/apache/incubator-gluten/issues/5009 for tracking this issue